src/databricks/labs/ucx/source_code/known.json

```diff
@@ -20614,6 +20614,130 @@
   "rich": {
     "rich": []
   },
+  "rpds-py": {
+    "rpds": []
+  },
+  "rpy2": {
+    "_rinterface_cffi_abi": [],
+    "rpy2": [],
+    "rpy2._rinterface_cffi_build": [],
+    "rpy2.interactive": [],
+    "rpy2.interactive.packages": [],
+    "rpy2.interactive.process_revents": [],
+    "rpy2.ipython": [],
+    "rpy2.ipython.ggplot": [],
+    "rpy2.ipython.html": [],
+    "rpy2.ipython.rmagic": [],
+    "rpy2.rinterface": [],
+    "rpy2.rinterface_lib": [],
+    "rpy2.rinterface_lib._rinterface_capi": [],
+    "rpy2.rinterface_lib.bufferprotocol": [],
+    "rpy2.rinterface_lib.callbacks": [],
+    "rpy2.rinterface_lib.conversion": [],
+    "rpy2.rinterface_lib.embedded": [],
+    "rpy2.rinterface_lib.embedded_mswin": [],
+    "rpy2.rinterface_lib.ffi_proxy": [],
+    "rpy2.rinterface_lib.memorymanagement": [],
+    "rpy2.rinterface_lib.na_values": [],
+    "rpy2.rinterface_lib.openrlib": [],
+    "rpy2.rinterface_lib.sexp": [],
+    "rpy2.rlike": [],
+    "rpy2.rlike.container": [],
+    "rpy2.rlike.functional": [],
+    "rpy2.rlike.indexing": [],
+    "rpy2.robjects": [],
+    "rpy2.robjects.constants": [],
+    "rpy2.robjects.conversion": [],
+    "rpy2.robjects.environments": [],
+    "rpy2.robjects.functions": [],
+    "rpy2.robjects.help": [],
+    "rpy2.robjects.language": [],
+    "rpy2.robjects.lib": [],
+    "rpy2.robjects.lib.dbplyr": [],
+    "rpy2.robjects.lib.dplyr": [],
+    "rpy2.robjects.lib.ggplot2": [],
+    "rpy2.robjects.lib.grdevices": [],
+    "rpy2.robjects.lib.grid": [],
+    "rpy2.robjects.lib.tidyr": [],
+    "rpy2.robjects.methods": [],
+    "rpy2.robjects.numpy2ri": [],
+    "rpy2.robjects.packages": [],
+    "rpy2.robjects.packages_utils": [],
+    "rpy2.robjects.pandas2ri": [],
+    "rpy2.robjects.robject": [],
+    "rpy2.robjects.vectors": [],
+    "rpy2.situation": [],
+    "rpy2.tests": [],
```

Comment:
`rpy2.tests` are irrelevant for the production code

Reply:
done